app/models/manageiq/providers/vmware/infra_manager/inventory/parser/host_system.rb

```diff
@@ -384,6 +384,18 @@ def parse_host_system_switches(host, props)
 
         persister_switch
       end
+
+      opaque_persister_switches = network[:opaqueSwitch].to_a.map do |switch|
+        uid = switch.key
+        persister.host_virtual_switches.build(
+          :host    => host,
+          :uid_ems => uid,
+          :name    => switch.name,
+          :type    => type
+        )
+      end
+
+      persister_switches.concat(opaque_persister_switches)
```

Comment:
do we use this array `persister_switches` somewhere?

Reply:
Yes they're passed in to the next method which refreshes the lans